src/linagora.esn.admin/app/modules/displayer/admin-modules-displayer.component.spec.js

```diff
@@ -130,7 +117,10 @@ describe('The adminModulesDisplayer component', function() {
         .trigger('input')
         .trigger('blur');
 
-      expect(element.find('admin-modules-save-button button').attr('disabled')).to.not.exist;
+      setTimeout(function(){
+        expect(element.find('admin-modules-save-button button').attr('disabled')).to.not.exist;
+        done()
+      }, 0)
```

Comment:
To avoid the `done` is not called when the `expect` is not satisfied and leads to an ambiguous error: `Timeout`. We need able to catch it.

```suggestion
      setTimeout(function(){
        try {
          expect(element.find('admin-modules-save-button button').attr('disabled')).to.not.exist;
          done();
        } catch (err) {
          done(err || new Error('should resolve'));
        }
      }, 0);
```

Reply:
Fixed